.pipelines/templates/PackageBuild.yml

```diff
@@ -182,6 +182,10 @@ steps:
         delta_fetch_arg="DELTA_FETCH=n"
       fi
 
+      if [[ -n "${{ parameters.maxCascadingRebuilds }}" ]]; then
+        max_cascading_rebuilds_arg="MAX_CASCADING_REBUILDS=${{ parameters.maxCascadingRebuilds }}"
```

Comment:
How is this limiting rebuilds to 1

Reply:
It's just not overwriting the default 1 rebuilt set when `QUICKBUILD_PACKAGE=y`. This change is a fix to how we pass the `MAX_CASCADING_REBUILDS` argument, so it doesn't interfere with other arguments. I'll re-word the summary and title.